tests/autoyast/installation.pm

```diff
@@ -16,6 +16,7 @@
 use strict;
 use base 'basetest';
 use testapi;
+use utils qw(clear_console);
 
```

Comment:
You're right if we talk about general perl (as used in os-autoinst and openQA). But for our tests, we tried hard to mimik a test DSL. Everything that is too specific to perl (as qw// :) you have to expect our test writers to just copy&paste. So give them something they can rely on to copy&paste as block to their test without having to understand -> import all utils at once.